fbw-a32nx/src/base/flybywire-aircraft-a320-neo/html_ui/Pages/VCockpit/Instruments/Airliners/FlyByWire_A320_Neo/FMC/A32NX_FMCMainDisplay.js

```diff
@@ -4599,19 +4473,18 @@ class FMCMainDisplay extends BaseAirliners {
         throw NXSystemMessages.formatError;
     }
 
+    /**
+     *
+     * @param {string} s
+     * @returns {boolean} true if valid place/bearing format
+     * @deprecated use {@link Fmgc.WaypointEntryUtils.isPbFormat}
+     */
     isPdFormat(s) {
+        return Fmgc.WaypointEntryUtils.isPdFormat(s);
         const pd = s.match(/^([^\/]+)\/([0-9]{1,3}(\.[0-9])?)$/);
         return pd !== null && this.isPlaceFormat(pd[1]);
```

Comment:
```suggestion
```

Dead code - but maybe remember to include ``const pd = s.match(/^([^\/]+)\/([0-9]{1,3}(\.[0-9])?)$/);`` this diff into WayPointEntryUtils isPdFormat.